src/rust/bitbox02-rust/src/shiftcrypto.bitbox02.rs

```diff
@@ -1543,6 +1547,35 @@ impl EthCoin {
         }
     }
 }
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, PartialOrd, Ord, ::prost::Enumeration)]
+#[repr(i32)]
+pub enum EthAddressCase {
+    Mixed = 0,
```

Comment:
nice, i don't know how prost decided to drop the long prefix, but it's very welcome :D

Reply:
yes, i also couldn't believe when I was looking at new enum field names 😄